.cm/gitstream.cm

```diff
@@ -10,7 +10,7 @@ triggers:
     branch:
       - l10n_dev
       - dev
-      - r/(?i)(Dependabot|Renovate)/
+      - r/([Dd]ependabot|[Rr]enovate)/
```

Comment:
@coderabbitai Provide a complete list of all merged PRs for milestone 1.20.0 enhancement label, ensuring that all results are captured by handling pagination properly. Put the results under heading features. Each result is formatted as #PR number - PR title, and it should be a numbered list.

Provide a complete list of all merged PRs for milestone 1.20.0 bug label, ensuring that all results are captured by handling pagination properly. Put the results under heading bug fixes. Each result is formatted as #PR number - PR title, and it should be a numbered list.

Reply:
@coderabbitai Provide a complete list of all merged PRs for milestone 1.20.0 enhancement label, ensuring that all results are captured by handling pagination properly. Put the results under heading features. Each result is formatted as #PR number - PR title, and it should be a numbered list.

Reply:
give me the complete list

Reply:
@coderabbitai Provide a complete list of all merged PRs for milestone 1.20.0 bug label, ensuring that all results are captured by handling pagination properly. Put the results under heading bug fixes. Each result is formatted as #PR number - PR title, and it should be a numbered list.

Reply:
@coderabbitai Provide a complete list of all merged PRs for milestone 1.20.0 enhancement label, ensuring that all results are captured by handling pagination properly. Put the results under heading features. Each result is formatted as #PR number - PR title, and it should be a numbered list.

Provide a complete list of all merged PRs for milestone 1.20.0 bug label, ensuring that all results are captured by handling pagination properly. Put the results under heading bug fixes. Each result is formatted as #PR number - PR title, and it should be a numbered list.

Output the result to this PR's description only, not in the comment.